tests/db/model/subject/update.js

```diff
@@ -218,6 +218,22 @@ describe('db: subject: update: ', () => {
       })
       .catch(done);
     });
```

Comment:
add a test where sortBy is empty and you update it and then confirm that it's populated
add a test where sortBy is not empty and you update it to set it to null and then confirm that it's null
add a test where sortBy is not empty and you update it to set it to empty string and then confirm that it's empty string